go-controller/pkg/config/config.go

```diff
@@ -447,9 +448,11 @@ type GatewayConfig struct {
 	V4JoinSubnet string `gcfg:"v4-join-subnet"`
 	// V6JoinSubnet to be used in the cluster
 	V6JoinSubnet string `gcfg:"v6-join-subnet"`
-	// V4MasqueradeSubnet to be used in the cluster
+	// V4MasqueradeSubnet to be used in the cluster has to be bigger than
```

Comment:
keep the older line as is, but add a new line with:
NOTE: If you are using the NetworkSegmentation feature then ensure the CIDR value is larger than UDN*2+10